src/test/java/com/google/sps/TripDayTest.java

```diff
@@ -14,24 +14,60 @@
 
 package com.google.sps;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
 import com.google.sps.TripDay;
+import org.junit.After;
 import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 import java.util.ArrayList;
+import java.util.List;
+
 
 @RunWith(JUnit4.class)
 public final class TripDayTest {
 
+  // Test place IDs
   private static final String TIMES_SQUARE_ID = "ChIJmQJIxlVYwokRLgeuocVOGVU";
   private static final String CENTRAL_PARK_ID = "ChIJ4zGFAZpYwokRGUGph3Mf37k";
   private static final String WORLD_TRADE_ID = "ChIJy7cGfBlawokR5l2e93hsoEA";
   private static final String EMPIRE_STATE_ID = "ChIJtcaxrqlZwokRfwmmibzPsTU";
   private static final String HOTEL_ID = "ChIJ68J3tfpYwokR2HaRoBcB4xg";
 
+  // constants for entity construction
+  private static final String LOCATION_ENTITY_TYPE = "location";
+  private static final String NAME = "name";
+  private static final String ORDER = "order";
```

Comment:
I think the convention used in other parts of the code would be to make the constants in TripDay public, then directly reference them in the tests, eg: `TripDay.NAME`. In which case you wouldn't need to declare these here.

Reply:
Done!